ovs-p4rt/sidecar/testing/geneve_encap_table_entry_test.cc

```diff
@@ -136,10 +137,14 @@ TEST_F(GeneveEncapTableEntryTest, encap_param_src_port_is_correct) {
     }
   }
 
-  // PrepareGeneveEncapTableEntry sets the dst_port parameter twice.
-  // The src_port parameter is not set at all. We get garbage instead.
-  ASSERT_TRUE(src_port.has_value());
-  // ASSERT_EQ(src_port.value(), SRC_PORT);
+  if (absl::GetFlag(FLAGS_check_src_port)) {
+    // The src_port field contains an arbitrary value that has nothing
```

Comment:
Perhaps we should add this comment to the actual source code file too in ovsp4rt.cc so that its evident that this is an intentional use by us.

Reply:
I agree.